src/core/ext/transport/binder/wire_format/wire_reader_impl.h

```diff
@@ -104,11 +104,20 @@ class WireReaderImpl : public WireReader {
   // called. Be cautious not to access it afterward.
   std::unique_ptr<Binder> other_end_binder_;
   absl::flat_hash_map<transaction_code_t, int32_t> expected_seq_num_;
+  absl::flat_hash_map<transaction_code_t, std::string> message_buffer_;
   std::unique_ptr<TransactionReceiver> tx_receiver_;
   bool is_client_;
   // When WireReaderImpl gets destructed, call on_destruct_callback_. This is
   // mostly for decrementing the reference count of its transport.
   std::function<void()> on_destruct_callback_;
+
+  // ACK every 16k bytes.
+  static constexpr size_t kFlowControlAckBytes = 16 * 1024;
+  size_t num_incoming_bytes_ = 0;
```

Comment:
How big is `size_t` in Android?

We probably want `int64_t` here to be safe. The wire format is already using 64 bits integers to represent size

Reply:
Done.